packages/editor/src/components/browser/CmsBrowser.tsx

```diff
@@ -140,7 +142,10 @@ const CmsBrowser = ({ value, onChange, noApiCall, typeFilter }: CmsBrowserProps)
 
   return (
     <>
-      <Checkbox label='Enable required Projects' value={requiredProject} onChange={() => setRequiredProject(!requiredProject)} />
+      <div className='cms-add-header'>
+        <Checkbox label='Enable required Projects' value={requiredProject} onChange={() => setRequiredProject(!requiredProject)} />
+        <Button icon={IvyIcons.Add} aria-label={'Create new CMS-String'} title={'Create new CMS-String'} onClick={() => newAction()} />
```

Comment:
Did you debug the backend code?
I think you need to add here some payload to the action, right @ivy-rew?

Reply:
yes; I accept the 'uri:String' of the currently selected element as parent container